introduction.md

```diff
@@ -1,3 +1,7 @@
 Hello, my name is Philippe. My favorite course path is Javascript.
-
+Deadly skills :
+* javascript,
+* git.
+* java,
+* C++
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?